crates/sui-indexer/src/schema/pg.rs

```diff
@@ -323,7 +325,9 @@ macro_rules! for_all_tables {
             packages,
             transactions,
             transactions_partition_0,
-            tx_calls,
```

Comment:
I'm a bit worried about the removal of this one. Does indexer reader use this for json rpc?

Reply:
Just in `indexer_reader` as a raw query, which I can address in this PR